pom.xml

```
@@ -397,7 +397,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/wallet-app/api/payment-wallet-for-ecommerce/v1/_openapi.json.tpl</inputSpec>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/pay-wallet-app/api/payment-wallet-for-ecommerce/v1/_openapi.json.tpl</inputSpec>
```

```suggestion
                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.327.0/src/domains/pay-wallet-app/api/payment-wallet-for-ecommerce/v1/_openapi.json.tpl</inputSpec>
```

Please set last release of infra in place of main